test/specs/svg.js

```diff
@@ -0,0 +1,44 @@
+const { fixture, FOCUS_RING_STYLE } = require('./helpers');
+const { Key, By } = require('selenium-webdriver');
+const expect = require('expect');
+const driver = global.__driver;
+
+// IE11 has a strange behavior where it will always focus an <svg> on the page.
+// This test is to verify that we don't hit an error in this situation.
+// See https://github.com/WICG/focus-visible/issues/80#issuecomment-383424156.
+describe('svg focus', function() {
+  beforeEach(function() {
+    return fixture('svg.html');
+  });
+
+  it('should NOT apply .focus-visible if a non-interactive SVG is keyboard focused', async function() {
+    async function tabUntil(body, end) {
+      let el;
+      while (true) {
+        focused = await driver.executeScript(`
```

Comment:
Why a template literal here?

Reply:
All of the tests that manually execute script through webdriver use a template literal. https://github.com/WICG/focus-visible/blob/master/test/specs/programmatic-focus-always.js#L32-L34

Reply:
Blargh, GitHub submitted my reply before I was finished typing.

I originally used template literals because the code was copied from another project and we had done it there. I can update all of the tests if you'd prefer that.